src/Sql/Sql/Database Backup/Cmdlet/GetAzureRmSqlDatabaseLongTermRetentionBackup.cs

```diff
@@ -169,6 +169,24 @@ public class GetAzureRmSqlDatabaseLongTermRetentionBackup : AzureSqlDatabaseLong
         [SupportsWildcards]
         public string BackupName { get; set; }
 
+        /// <summary>
+        /// Gets or sets the name of the resource group to use.
+        /// </summary>
+        [Parameter(Mandatory = false,
+            ParameterSetName = LocationSet,
+            Position = 0,
+            HelpMessage = "The name of the resource group.")]
+        [Parameter(Mandatory = false,
+            ParameterSetName = ServerNameSet,
+            Position = 0,
+            HelpMessage = "The name of the resource group.")]
+        [Parameter(Mandatory = false,
+            ParameterSetName = BackupNameSet,
+            Position = 0,
```

Comment:
Do not add a position to an optional parameter, and certainly not position 0

Reply:
fixed.